### ISSUE TITLE
a bug? : is the "<=" in line 195 supposed to be ">" ?

### ISSUE THREAD

**Issue by chenkq7** [ejones313/roben#4]
https://github.com/ejones313/roben/blob/69b613d6b6c7bdf3832456d4b38f71d923e23e73/attacks.py#L195

**Comment by ejones313** [comment]
Yes, good catch, updated! Fortunately, we did not use this experiment in the paper, so all the numbers there are still accurate :)
